From b6520696b34d4847b0cc2f121e6a226e0d32fcb3 Mon Sep 17 00:00:00 2001
From: Kyle Repinski <repinski23@gmail.com>
Date: Sat, 17 Sep 2016 10:09:17 -0500
Subject: [PATCH 18/20] domx: Ensure DOMX_TUNA is defined everywhere.

Probably not necessary *everywhere*, but this has caused enough headaches.

Change-Id: I4941048d1b28ca4558d55e3eca4012dbe47145bb
---
 domx/domx/Android.mk                | 2 +-
 domx/domx/memmgr_rpc/src/Android.mk | 4 ++++
 domx/mm_osal/Android.mk             | 4 ++++
 domx/omx_core/Android.mk            | 4 ++++
 4 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/domx/domx/Android.mk b/domx/domx/Android.mk
index 52d19e2..59d02b1 100644
--- a/domx/domx/Android.mk
+++ b/domx/domx/Android.mk
@@ -48,7 +48,7 @@ LOCAL_C_INCLUDES += $(HARDWARE_TI_OMAP4_BASE)/libion
 endif
 
 ifeq ($(TARGET_BOOTLOADER_BOARD_NAME),tuna)
-LOCAL_CFLAGS += -DDOMX_TUNA
+	LOCAL_CFLAGS += -DDOMX_TUNA
 endif
 
 LOCAL_MODULE:= libdomx
diff --git a/domx/domx/memmgr_rpc/src/Android.mk b/domx/domx/memmgr_rpc/src/Android.mk
index b71e9d5..2eb9710 100755
--- a/domx/domx/memmgr_rpc/src/Android.mk
+++ b/domx/domx/memmgr_rpc/src/Android.mk
@@ -18,6 +18,10 @@ LOCAL_C_INCLUDES += \
 
 LOCAL_CFLAGS += -D_Android
 
+ifeq ($(TARGET_BOOTLOADER_BOARD_NAME),tuna)
+	LOCAL_CFLAGS += -DDOMX_TUNA
+endif
+
 LOCAL_SHARED_LIBRARIES := \
 	libOMX_CoreOsal \
 	libipcutils \
diff --git a/domx/mm_osal/Android.mk b/domx/mm_osal/Android.mk
index f41f131..aa9e62a 100644
--- a/domx/mm_osal/Android.mk
+++ b/domx/mm_osal/Android.mk
@@ -24,6 +24,10 @@ LOCAL_SHARED_LIBRARIES := \
 LOCAL_CFLAGS += -DOMAP_2430 -DOMX_DEBUG -D_Android -D_POSIX_VERSION_1_
 LOCAL_CFLAGS += -DTIMM_OSAL_DEBUG_TRACE_DETAIL=1 # quiet
 
+ifeq ($(TARGET_BOOTLOADER_BOARD_NAME),tuna)
+	LOCAL_CFLAGS += -DDOMX_TUNA
+endif
+
 LOCAL_MODULE:= libmm_osal
 LOCAL_MODULE_TAGS:= optional
 
diff --git a/domx/omx_core/Android.mk b/domx/omx_core/Android.mk
index 3308e79..813467d 100644
--- a/domx/omx_core/Android.mk
+++ b/domx/omx_core/Android.mk
@@ -20,6 +20,10 @@ ifdef USE_ITTIAM_AAC
 LOCAL_CFLAGS += -DUSE_ITTIAM_AAC
 endif
 
+ifeq ($(TARGET_BOOTLOADER_BOARD_NAME),tuna)
+LOCAL_CFLAGS += -DDOMX_TUNA
+endif
+
 LOCAL_CFLAGS += -DSTATIC_TABLE -D_Android -DCHECK_SECURE_STATE
 LOCAL_MODULE:= libOMX_Core
 LOCAL_MODULE_TAGS:= optional
-- 
2.7.4

